src/behaviors/introductions/introductions.js

```diff
@@ -12,14 +12,13 @@ class Introductions extends Behavior {
     super.initialize(bot);
     let promise = Promise.resolve();
 
-    Promise.all([bot.getUser('imjoshdean'), bot.getUser('sheva')]).then((users) => {
-      const [josh, sheva] = users,
-        joshTag = `<@${josh.id}|${josh.name}>`,
-        shevaTag = `<@${sheva.id}|${sheva.name}>`;
+    Promise.all([bot.getUser('chibishibe')]).then((users) => {
```

Comment:
I'd like to ask you to remove these changes. You are welcome to keep them local, but I'd like to preserve this and the `messages.json` file on my own repository.

Reply:
Would it be desirable to merge all contributors so far, or just keep it at your version?
Also, even if it is left at just the two of you, could we do some fancy map/reduce to make it so there's not a variable per contributor? :)